src/components/ConnectWallet/Wallets/ElectrumAndroid.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <card-widget
+    header="Here's how to connect Electrum Wallet (Android) to your Umbrel"
+  >
+    <div class="px-3 px-lg-4 pb-3">
+      <ol class="connect-wallet-steps">
+        <li class="connect-wallet-step">
+          Download
+          <b-link
+            href="https://play.google.com/store/apps/details?id=org.torproject.android"
```

Comment:
Removed F-Droid to again keep things simple. Users who use F-Droid will probably use it anyway to download Orbot.